packages/core/src/domain/session/sessionStore.ts

```diff
@@ -123,9 +130,30 @@ export function startSessionStore<TrackingType extends string>(
     return sessionState
   }
 
+  function startSession() {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I found confusing that `startSession` uses `getExpiredSessionState()` to start the session. Maybe a comment or another name like `initSession` could help 🤷‍♂️?

Reply:
`initializeSession` was confusing too. see this comment: https://github.com/DataDog/browser-sdk/pull/2673#discussion_r1568922841

We could rename `NotStarted` to `NotInitialized` and `startSession()` to `initializeSession` could help. However IMHO `init` and `start` are synonyms and it will feel similarly confusing to use `getExpiredSessionState()` here.

I think the state diagrams in the doc should helps to understand that a session starts (or is initialized) in the `Expired` state:

```mermaid
stateDiagram-v2
state fork_state <<fork>>
state fork_state2 <<fork>>

[*] --> fork_state

fork_state --> NotStarted
fork_state --> Expired
fork_state --> Tracked
fork_state --> NotTracked
NotStarted --> Expired: startSession()

Expired --> fork_state2
Tracked --> fork_state2
NotTracked --> fork_state2
fork_state2 --> [*]: expandOrRenew()
```

Reply:
Fortunately the diagrams are very cool 🙂  Fine by me!